website/docs/explorer/getting-started.mdx

```diff
@@ -35,7 +35,20 @@ metadata:
 spec:
   # ... other spec components
   values:
-    enableExplorer: true
+  explorer:
```

Comment:
given its a getting started page, a minimal configuration would be better 

```
explorer:
  enabled: true # global enable/disable flag
```
and then leave other cases for the user to go onto the configuration page 

https://docs.gitops.weave.works/docs/explorer/configuration/

where to give a little more guidance and the configuration options that we provide 

https://docs.gitops.weave.works/docs/explorer/configuration/#setup